Exporter/Exporter.php

```diff
@@ -14,13 +14,51 @@
 use Exporter\Source\SourceIteratorInterface;
 use Exporter\Writer\CsvWriter;
 use Exporter\Writer\JsonWriter;
+use Exporter\Writer\WriterInterface;
 use Exporter\Writer\XlsWriter;
 use Exporter\Writer\XmlWriter;
 use Symfony\Component\HttpFoundation\StreamedResponse;
 
+/**
+ * NEXT MAJOR : make this class final.
```

Comment:
And this is not related to the PR i think. Would be better to have an issue for that.